rio_toa/toa_utils.py

```diff
@@ -141,3 +140,18 @@ def temp_rescale(arr, temp_scale):
     else:
         raise ValueError('%s is not a valid temperature scale'
                          % (temp_scale))
+
+
+def normalize_scale(rescale_factor, dtype):
+    default_scales = {
+        'uint8': 215,
+        'uint16': 55000,
+        'float32': 1.0}
```

Comment:
@perrygeo Since we're setting clip as True, how about we use the full range (higher value) and add a warning message when we have an overflow?